srcpkgs/cpio/template

```diff
@@ -1,13 +1,18 @@
 # Template file for 'cpio'
 pkgname=cpio
 version=2.12
-revision=2
+revision=3
 build_style=gnu-configure
 configure_args="--with-rmt=/usr/bin/rmt"
+hostmakedepends="automake gettext-devel"
 checkdepends="bash"
 short_desc="GNU copy-in/out (cpio) with remote magnetic tape (rmt) support"
 maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-3.0-or-later"
 homepage="http://www.gnu.org/software/cpio/"
 distfiles="${GNU_SITE}/cpio/cpio-${version}.tar.bz2"
 checksum=70998c5816ace8407c8b101c9ba1ffd3ebbecba1f5031046893307580ec1296e
+
+pre_configure() {
+	autoreconf -if
```

Comment:
do we need to regenerate the configure files ?

Reply:
Unfortunately if we don’t, it gives autoconf/automake error